04_environment/design.tex

```diff
@@ -0,0 +1,289 @@
+
+importance of prevention and incentivise islands to contribute to the Common Pool is highlighted, thus if the amount of resources in the common pool exceeds the threshold when the disaster happens, then the “power” of disaster towards the islands and common pool will be decreased. It makes sense in real life, as more preparation will result in less damage. The actual damage of disaster is decreased by half.
+
+\subsection{Example Distribution}
+This can be described by the examples in Table~\ref{tab:Demonstration of how the common pool work in different scenarios}, which shows 4 possible scenario:
+
+\begin{itemize}
+    \item Contribution to common pool does not meet threshold and common pool cannot fully mitigate disaster’s damage.
+    \item Contribution to common pool does not meet threshold but common pool can fully mitigate disaster’s damage.
+    \item Contribution to common pool meets threshold but common pool cannot fully mitigate disaster’s damage.
+    \item Contribution to common pool meets threshold and common pool can fully mitigate disaster’s damage.
+\end{itemize}
+
+\begin{table}[!htb]
+\begin{center}
+\begin{tabular}{|l|c|c|c|c|}
+\hline
+                               & \textbf{Scenario 1} & \textbf{Scenario 2} & \textbf{Scenario 3} & \textbf{Scenario 4} \\ \hline
+\textbf{Disaster Total Damage} & 1000                & 200                 & 1200                & 1200                \\ \hline
+\textbf{CP's Threshold}        & 500                 & 500                 & 500                 & 500                 \\ \hline
+\textbf{CP's Current Value}    & 300                 & 300                 & 500                 & 700                 \\ \hline
+\textbf{CP's Multiplier}       & 1                   & 1                   & 0.5                 & 0.5                 \\ \hline
+\textbf{Mitigated Damage}      & 300                 & 200                 & 500                 & 1200                \\ \hline
+\textbf{Remaining Damage}      & 700                 & 0                   & 100                 & 0                   \\ \hline
+\textbf{CP's Remaining Value}  & 0                   & 100                 & 0                   & 100                 \\ \hline
+\end{tabular}
+\end{center}
+\caption{Demonstration of how the common pool work in different scenarios}
+\label{tab:Demonstration of how the common pool work in different scenarios}
+\end{table}
+
+In scenario 1, the disaster total damage is high. However, there is no preparation from the islands through contribution to the common pool. As a result, the common pool could not mitigate all of the damage and $70\%$ of the damage will be directly on the islands.
```

Comment:
these four scenarios might be nice to present in an itemised list 

up to you nitpicking here